src/Nethermind/Nethermind.BeaconNode.Host/Nethermind.BeaconNode.Host.csproj

```diff
@@ -24,5 +24,14 @@
     <ProjectReference Include="..\Nethermind.BeaconNode\Nethermind.BeaconNode.csproj" />
     <ProjectReference Include="..\Nethermind.BeaconNode.Storage\Nethermind.BeaconNode.Storage.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Content Remove="configuration\**" />
```

Comment:
was it planned?

Reply:
Planned to not be included in the output? Yes. 

The configuration folder in that project has the YAML files from the specs., and a powershell script that converts them to JSON snippets, that I then paste into appsettings.json. The appsettings files are used in the published version, not the YAML.

The intention is that the appsettings files are standard for .NET Core and will have examples for third party apps. e.g. Serilog has config examples in JSON. (The actual config system won't care, command line, environment, etc.; it is more for a dev familiar with .NET).

Note that the underlying code also supports the YAML file, if that is wanted. Either will work (or command line, environment, etc).